x/crosschain/simulation/operation_migrate_erc20_custody_funds.go

```diff
@@ -0,0 +1,123 @@
+package simulation
+
+import (
+	"math/rand"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	moduletestutil "github.com/cosmos/cosmos-sdk/types/module/testutil"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+
+	"github.com/zeta-chain/node/pkg/chains"
+	"github.com/zeta-chain/node/testutil/sample"
+	"github.com/zeta-chain/node/x/crosschain/keeper"
+	"github.com/zeta-chain/node/x/crosschain/types"
+)
+
+// SimulateMigrateERC20CustodyFunds generates a MsgMigrateERC20CustodyFunds with random values and delivers it
+func SimulateMigrateERC20CustodyFunds(k keeper.Keeper) simtypes.Operation {
```

Comment:
Message will be removed since we migrated to contracts v2, let's remove this one